documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -63,32 +114,77 @@ spec:
+=== Topic creation group configuration properties
+The `default` topic creation group and each custom group is associated with a unique set of configuration properties.
+You can configure a group to include any of the {link-kafka-docs}/#topicconfigs[Kafka topic-level configuration properties].
+For example, you can specify the {link-kafka-docs}/#topicconfigs_cleanup.policy[cleanup policy for old topic segments], {link-kafka-docs}/#topicconfigs_retention.ms[retention time], or the {link-kafka-docs}/#topicconfigs_compression.type[topic compression type] for a topic group.
+You must define at least a minimum set of properties to describe the configuration of the topics to be created.
+
+If no custom groups are registered, or if the `include` patterns for any registered groups don't match the names of any topics to be created,
+then Kafka Connect uses the configuration of the `default` group to create topics.
 
 [NOTE]
 ====
-You can specify both, `include` and `exclude` parameters, but note that exclusion rules
-take precedence and override any inclusion rules for topics.
+If you specify both `include` and `exclude` properties for a custom topic group, the exclusion rules
+take precedence, and override the inclusion rules.
 ====
 
-You don't have to specify any custom group. When there's no custom group registered or
-the registered group's `include` patterns don't match the topic which is to be created
-then the default config will be used.
-
-You can specify all {link-kafka-docs}/#topicconfigs[topic level configuration parameters]
-to customize how topics will be created.
 
-See {link-prefix}:{link-install-debezium}#_configuring_debezium_topics[Configuring {prodname} Topics] section in the
+ifdef::community[]
+See {link-prefix}:{link-install-debezium}#_configuring_debezium_topics[Configuring {prodname} topics] in the
 {prodname} installation guide on generic topic configuration considerations.
+endif::community[]
 
+ifdef::product[]
+For general information about configuring topics, see {link-prefix}:{LinkDebeziumInstallOpenShift}#kafka-topic-creation-recommendations[Kafka topic creation recommendations] in {NameDebeziumInstallOpenShift}.
+endif::product[]
+
+// Type: procedure
+// ModuleID: specifying-the-configuration-for-the-debezium-default-topic-creation-group
+// Title: Specifying the configuration for the {prodname} default topic creation group
+[id="default-topic-creation-group-configuration"]
 === Default group configuration
 
-The default config can be passed in the connector config JSON like:
+You must create a default topic creation group and define a configuration for it.
+The configuration for the default topic creation group is applied to any topics whose names do not match the `include` list pattern of a custom topic creation group.
+
+.Procedure
 
+* To define properties for the `topic.creation.default` group, add them to the connector configuration JSON, as shown in the following example:
```

Comment:
For downstream, this shouldn't be JSON but rather the AMQ Streams CR.

Reply:
> AMQ Streams CR.

I think that what you mean is that on Streams, you must define the properties for the default group in the CR _for the Debezium connector_? Is that correct?

I'd like to provide an example, but I'm having trouble coming up with one that I can model. I believe that you would specify the properties in `spec.config`, yes? So something like this:

```
spec:
...

   config:
...
     topic.creation.default.replication.factor: 3 
     topic.creation.default.partitions: 10  
     topic.creation.default.cleanup.policy: compact  
     topic.creation.default.compression.type: lz4  
```

Reply:
Added a conditionalized example for downstream that shows the configuration as an excerpt from a connector custom resource.